src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/ConnectionHandlerContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.IO;
+using Microsoft.Data.SqlClient.SNI;
+using Microsoft.Data.SqlClientX.Handlers;
+
+namespace Microsoft.Data.SqlClient.Microsoft.Data.SqlClientX.Handlers
+{    /// <summary>
+     /// Class that contains all context data needed for various handlers.
+     /// </summary>
+    internal class ConnectionHandlerContext : HandlerRequest
+    {
+        /// <summary>
+        /// Class that contains data required to handle a connection request.
+        /// </summary>
+        public String connectionString { get; set; }
```

Comment:
Property names start with Caps 

```suggestion
        public String ConnectionString { get; set; }
```